google-cloud-pubsub/samples/pubsub_subscriber_exactly_once_delivery.rb

```diff
@@ -16,8 +16,8 @@
 require "google/cloud/pubsub"
 
 # Shows how to register callback to acknowledge method and access the result passed in
-class PubsubSubscriberExactlyOnce
-  def subscriber_exactly_once project_id:, topic_id:, subscription_id:
+class PubsubSubscriberExactlyOnceDelivery
+  def subscriber_exactly_once_delivery project_id:, topic_id:, subscription_id:
```

Comment:
Can you add a comment here that for non exactly once delivery subs, acknowledge will always return success?